sdformat_to_mjcf/sdformat_to_mjcf/converters/joint.py

```diff
@@ -66,7 +66,8 @@ def add_joint(body, joint):
     :rtype: mjcf.Element
     """
     if joint is None:
-        return body.add("freejoint", name="freejoint")
+        return body.add("freejoint",
+                        name=su.prefix_name(body.name, "freejoint"))
```

Comment:
`body.name` might is optinal, it might be `None`, you should check this first

Reply:
`prefix_name` checks if it's `None` though.